concept/type_/entity_type.rs

```diff
@@ -239,6 +238,14 @@ impl<'a> OwnerAPI<'a> for EntityType<'a> {
         let expected_owns = Owns::new(ObjectType::Entity(self.clone().into_owned()), attribute_type);
         Ok(self.get_owns(snapshot, type_manager)?.contains(&expected_owns).then_some(expected_owns))
     }
+
+    fn get_owns_transitive<'m, Snapshot: ReadableSnapshot>(
```

Comment:
Let's make these all take an enum `Transitivity` which is `DECLARED` or `TRANSITIVE` and not have two method types throughout